packages/sdk/src/modules/did/index.ts

```diff
@@ -34,16 +52,49 @@ interface CreateDidOptions {
   customDocumentFields?: CustomDocumentFields;
 }
 
+interface ReadDidOptions {
+  name: string;
+  address?: Address;
+}
+
+interface UpdateDidOptions {
+  name: string;
+  address?: Address;
+  seed?: string;
+  customDocumentFields: UpdateDocumentFields;
+}
+
+interface UpdateDidDocumentOptions {
+  didAccountAddress: Address;
+  didControllerAddress: Address;
+  customDocumentFields?: UpdateDocumentFields;
+  oldDocument: DidDocument
```

Comment:
It is used to pass the previously read DID document to the _updateDidDocument function  (which is a private function) so it can use the previous doc to update the necessary fields.